src/getting-started/build-a-sample-app.md

```diff
@@ -61,6 +61,10 @@ impl Component for Model {
             </div>
         }
     }
+
+    fn change(&mut self, _props: Self::Properties) -> ShouldRender {
```

Comment:
nit: I usually prefer to keep `fn change()` above `fn view()`. Could you also add a comment above `false` saying that `change` should only return true if new properties are received and this component has no properties?

Reply:
Fixed